### DISCUSSION TITLE
lximage-qt: how to make a screenshot

### DISCUSSION THREAD

**Discussion by DavidPerezIngeniero** [lxqt/lxqt#2614] [Media]
I'm using Plasma 6.1.15 and don't see how to capture a screenshot.
Reviewed all the manu items and toolbar icons and read the doc.
https://github.com/lxqt/lximage-qt

**Comment by tsujan** [comment]
> I'm using Plasma 6.1.15 and don't see how to capture a screenshot.

If you use Wayland, you can't. The menu item "Capture Screenshot" is visible under the "File" menu only on X11.

On Wayland there isn't a universal way of taking screenshots. Each compositor may have its own tool. Plasma has Spectacle.

**Comment by Bluey26** [comment]
Hello, to use lximage-qt to capture screenshots, you need to be in X11. (it does not seem to work in Wayland)

I just have tried to capture a screenshow. The command is: `lximage-qt -s`

You can also set a keyboard shortcut, (Edit>Preferences>Shortcut --> Capture Screenshot) so you can take screenshots when the app is open.

If you are in Wayland (Plasma is wayland by default now, as far as i know), you could rely in other methods, because both `lximage-qt -s` and `screengrab -m` dont work in Wayland at the moment.

For example, there is ` grim -g "$(slurp)" /tmp/screen_$(date +'%a_%T_.png')` if you want to select a screen area.

**Comment by DavidPerezIngeniero** [comment]
Thanks for your valuable answer!
